source/mainnet/net/guides/developer-page.rst

```diff
@@ -22,6 +22,8 @@ Official resources
 
     - `CIS2 token standard <https://proposals.concordium.software/CIS/cis-2.html>`_ is applicable for Smart Contracts v1.
 
+- See :ref:`How to create proofs for dApps and services <create-proofs>` for information about how to write proofs that interact with Concordium wallets
```

Comment:
I don't think you ever write proofs. The wallet produces proofs. What you write are statements, and the wallet produces proofs.